src/index.html

```diff
@@ -4,9 +4,254 @@
+          </div>
+          <a class="buy-link"
+              href="#"
+              >КУПИТЬ
+          </a>
+        </div>
+
+        <div class="product-card">
+          <img
+            src="/images/imac.jpeg"
+            class="product-card-image"
+            alt="item">
+          <h2 class="product-card-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+          <p class="product-card-code">Код товара: 195434</p>
+          <div class="product-card-rate">
+            <div class="stars-container">
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+            </div>
+            Отзывов: 5
+          </div>
+          <div class="product-card-price">
+            <span class="price">Цена:</span> <span class="price-value">69 999 грн</span>
+          </div>
+          <a class="buy-link"
+              href="#"
+              >КУПИТЬ
+          </a>
+        </div>
+
+        <div class="product-card">
+          <img
+            src="/images/imac.jpeg"
+            class="product-card-image"
+            alt="item">
+          <h2 class="product-card-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+          <p class="product-card-code">Код товара: 195434</p>
+          <div class="product-card-rate">
+            <div class="stars-container">
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+            </div>
+            Отзывов: 5
+          </div>
+          <div class="product-card-price">
+            <span class="price">Цена:</span> <span class="price-value">69 999 грн</span>
+          </div>
+          <a class="buy-link"
+              href="#"
+              >КУПИТЬ
+          </a>
+        </div>
+
+        <div class="product-card">
+          <img
+            src="/images/imac.jpeg"
+            class="product-card-image"
+            alt="item">
+          <h2 class="product-card-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+          <p class="product-card-code">Код товара: 195434</p>
+          <div class="product-card-rate">
+            <div class="stars-container">
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+            </div>
+            Отзывов: 5
+          </div>
+          <div class="product-card-price">
+            <span class="price">Цена:</span> <span class="price-value">69 999 грн</span>
+          </div>
+          <a class="buy-link"
+              href="#"
+              >КУПИТЬ
```

Comment:
```suggestion
          <a 
            class="buy-link"
            href="#"
          >
            Купить
```
Делай текст uppercase через css